SparkPost/2web2ui
# SA-1548: Report Builder table adds filter when clicking on resource

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
 - Added AddFilterLink component.
 - Made the first column of the breakdown table a link to add a filter

### How To Test
 - Enable Hibana
 - Go to /signals/analytics
 - Under `Break Down By`, select any filter type. 
 - Click any filter and verify that it is added as a filter.
 - Also verify that right clicking and middle/cmd clicking works as well.

### To Do
- [x] Maybe Cypress test?
